test/integrationtests/username_password_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package integrationtests
+
+import (
+	"context"
+	"testing"
+
+	"github.com/AzureAD/microsoft-authentication-library-for-go/msal"
+)
+
+var labClientInstance = &labClient{}
```

Comment:
Not a fan of the globals for tests.  They usually end up with weird side effects later.  I'm recommending a single test below that is table drive, this can move inside that test.